concent_api/core/models.py

```diff
@@ -684,3 +686,54 @@ class PendingEthereumTransaction(Model):
     s = DecimalField(max_digits=78, decimal_places=0)
 
     created_at = DateTimeField(auto_now_add=True)
+
+
+class DepositAccount(Model):
+    client = ForeignKey(Client)
+    ethereum_public_key = CharField(max_length=ETHEREUM_ADDRESS_LENGTH)
+    created_at = DateTimeField(auto_now_add=True)
+
+    def clean(self) -> None:
+        if not isinstance(self.ethereum_public_key, str) or len(self.ethereum_public_key) != 42:
```

Comment:
Do `super().clean()` too here and below.

Reply:
Done.

Reply:
Please use `core.constants.ETHEREUM_ADDRESS_LENGTH`.

Reply:
Done.